src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+import { Person } from '../../types';
+import { getPreparedPerson } from '../../utils/getPreparedPeople';
+import { PeopleFilters } from '../PeopleFilters/PeopleFilters';
+import { filter } from '../../utils/filter';
+import { useSearchParams } from 'react-router-dom';
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
+  const [peopleList, setPeopleList] = useState<Person[]>([]);
+  const [selectedFilter, setSelectedFilter] = useSearchParams();
+  const [hasSideBarVisible, setHasSideBarVisible] = useState<boolean>(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    setHasSideBarVisible(false);
+
+    getPeople()
+      .then(result => {
+        setPeopleList(result);
+        setHasSideBarVisible(true);
+      })
+      .catch(() => setIsError(true))
+      .finally(() => {
+        setIsLoading(false);
+      });
+
+    return () => {
+      setPeopleList([]);
+    };
+  }, []);
+
+  const preparedPeople = getPreparedPerson(peopleList);
+  const filteredPeople = filter(selectedFilter, preparedPeople);
+
+  return (
+    <div data-cy="app">
+      <main className="section">
+        <div className="container">
+          <h1 className="title">People Page</h1>
+          <div className="block">
+            <div className="columns is-desktop is-flex-direction-row-reverse">
+              <div className="column is-7-tablet is-narrow-desktop">
+                {hasSideBarVisible && !isError && !isLoading && (
+                  <PeopleFilters setSelectedFilter={setSelectedFilter} />
+                )}
+              </div>
+              <div className="column">
+                <div className="box table-container">
+                  {isLoading && <Loader />}
+                  {isError && (
+                    <p data-cy="peopleLoadingError" className="has-text-danger">
+                      Something went wrong
+                    </p>
+                  )}
+                  {!peopleList.length && !isLoading && !isError && (
+                    <p data-cy="noPeopleMessage">
+                      There are no people on the server
+                    </p>
+                  )}
+                  {!!filteredPeople.length ? (
+                    <PeopleTable people={filteredPeople} />
+                  ) : (
+                    <p>
+                      There are no people matching the current search criteria
+                    </p>
+                  )}
+                </div>
+              </div>
+            </div>
+          </div>
```

Comment:
change indents in all the cases for better readability
```suggestion
          <h1 className="title">People Page</h1>
          
          <div className="block">
            <div className="columns is-desktop is-flex-direction-row-reverse">
              <div className="column is-7-tablet is-narrow-desktop">
                {hasSideBarVisible && !isError && !isLoading && (
                  <PeopleFilters setSelectedFilter={setSelectedFilter} />
                )}
              </div>
              
              <div className="column">
                <div className="box table-container">
                  {isLoading && <Loader />}
                  
                  {isError && (
                    <p data-cy="peopleLoadingError" className="has-text-danger">
                      Something went wrong
                    </p>
                  )}
                  
                  {!peopleList.length && !isLoading && !isError && (
                    <p data-cy="noPeopleMessage">
                      There are no people on the server
                    </p>
                  )}
                  
                  {!!filteredPeople.length ? (
                    <PeopleTable people={filteredPeople} />
                  ) : (
                    <p>
                      There are no people matching the current search criteria
                    </p>
                  )}
                </div>
              </div>
            </div>
          </div>
```

Reply:
done